### PR TITLE
Added New TabBar Component named BEKCurveTabbar 

### PR DESCRIPTION
**Pull request by behrad-kzm**
[New Component]
- New UITabbar Custom Component added to the Tab Bar Section.
 compatible with XCode +10 and fully customizable via Interface_Builder panel. BEKCurveTabBar derived the UITabBar class and compatible with every iOS device.

## Project URL

https://github.com/behrad-kzm/BEKCurveTabbar

## Category

Tab Bar

## Description

just added a new component in right formating with 2 lines of description about customizability and Compatibility with Xcode Interface builder

## Why it should be included to `awesome-ios` (mandatory)

There is no Tab bar in awesome-ios that developer can change the appearance realtime in interface builder at this level of customizability. 
also it derived UITabBarController and UITabBar so it will stay compatible with every iOS Environment.
BEKCurveTabbar has different Appearances in iphone X screen types.
## Checklist

- [ ] Has 50 GitHub stargazers or more
- [x] Only one project/change is in this pull request
- [x] Isn't an archived project
- [ ] Has more than one contributor
- [ ] Has unit tests, integration tests or UI tests
- [ ] Addition in chronological order (bottom of category)
- [ ] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 4 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 16 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
4 | [302](https://httpstatuses.com/302) | https://triplebyte.com/a/azelHFa/d redirects to<br>https://triplebyte.com/
276 | [301](https://httpstatuses.com/301) | https://github.com/analytics-pros/Swift-GA-Tracker-for-Apple-tvOS redirects to<br>https://github.com/adswerve/Swift-GA-Tracker-for-Apple-tvOS
474 | [404](https://httpstatuses.com/404) | https://github.com/czater/Colorify 
767 | [301](https://httpstatuses.com/301) | https://github.com/igormatyushkin014/Sensitive redirects to<br>https://github.com/hellowizman/Sensitive
935 | [301](https://httpstatuses.com/301) | https://github.com/onmyway133/Anchors redirects to<br>https://github.com/onmyway133/EasyAnchor
1333 | [404](https://httpstatuses.com/404) | https://github.com/jorik041/PerfectAPIClient 
1382 | [301](https://httpstatuses.com/301) | http://www.catapush.com/ redirects to<br>https://www.catapush.com/
1452 | [301](https://httpstatuses.com/301) | https://github.com/adrianmateoaea24/magic-mapper-swift redirects to<br>https://github.com/adrianmteo/magic-mapper-swift
1532 | [301](https://httpstatuses.com/301) | https://github.com/IvanVorobei/SPPermission redirects to<br>https://github.com/ivanvorobei/SPPermissions
1615 | [301](https://httpstatuses.com/301) | https://github.com/facebook/facebook-swift-sdk redirects to<br>https://github.com/facebookarchive/facebook-swift-sdk
2010 | [301](https://httpstatuses.com/301) | https://github.com/Babylonpartners/DrawerKit redirects to<br>https://github.com/babylonhealth/DrawerKit
2097 | [301](https://httpstatuses.com/301) | https://github.com/soheil/RadialLayer redirects to<br>https://github.com/soheil-zz/RadialLayer
2138 | [301](https://httpstatuses.com/301) | https://github.com/ML-Works/Overlap redirects to<br>https://github.com/MLWOS/Overlap
2505 | [301](https://httpstatuses.com/301) | https://github.com/rynecheow/PopupKit redirects to<br>https://github.com/Pointwelve/PopupKit
2548 | [301](https://httpstatuses.com/301) | https://github.com/OpenFeyn/KafkaRefresh redirects to<br>https://github.com/HsiaohuiHsiang/KafkaRefresh
3253 | [301](https://httpstatuses.com/301) | http://jamesonquave.com/blog/tutorials/ redirects to<br>https://jamesonquave.com/blog/tutorials/


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by behrad-kzm** [comment]
added new Files to use the component without interface builder and an example of usage

File: README.md [hunk 1]
```diff
@@ -2635,6 +2635,7 @@ Most of these are paid services, some have free tiers.
 - [Switch](https://github.com/T-Pham/Switch) - An iOS switch control implemented in Swift with full Interface Builder support.
 
 ### Tab Bar
+- [BEKCurveTabbar](https://github.com/behrad-kzm/BEKCurveTabbar) - compatible with XCode +10 and fully customizable via Interface_Builder panel. BEKCurveTabBar derived UITabBar class and compatible with every iOS devices.
```
Comment (lfarah) [inline README.md]: Please move to the end of the Tab Bar list

File: README.md [hunk 2]
```diff
@@ -2670,6 +2671,7 @@ Most of these are paid services, some have free tiers.
 - [MGSwipeTableCell](https://github.com/MortimerGoro/MGSwipeTableCell) - UITableViewCell subclass that allows to display swippable buttons with a variety of transitions.
 - [YXTPageView](https://github.com/hanton/YXTPageView) - A PageView, which supporting scrolling to transition between a UIView and a UITableView.
 - [ConfigurableTableViewController](https://github.com/fastred/ConfigurableTableViewController) - Typed, yet Flexible Table View Controller https://holko.pl/2016/01/05/typed-table-view-controller/
+- [BEKMultiCellTable](https://github.com/behrad-kzm/BEKMultiCellTable) - A generic table view based on MVVM that allows you to add multiple cell types at runtime in a single line of code.
```
Comment (lfarah) [inline README.md]: Please open a separate PR for BEKMultiCellTable

**Comment by vsouza** [comment]
Thanks for contributing!

**Pull request merged**